ckanext/datarequests/auth.py

```diff
@@ -17,13 +17,26 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with CKAN Data Requests Extension. If not, see <http://www.gnu.org/licenses/>.
 
-from ckan.plugins.toolkit import auth_allow_anonymous_access, get_action
+from ckan import authz
+from ckan.plugins.toolkit import asbool, auth_allow_anonymous_access, config, get_action
 
 from . import constants
 
 
 def create_datarequest(context, data_dict):
-    return {'success': True}
+    return {
+        'success': asbool(config.get("ckanext.auth.create_datarequest_if_not_in_organization", "True"))
```

Comment:
Does the "True" here mean that it defaults to True if the config doesn't get a result? If so, would it be better to default to False in this case as we are trying to be more restrictive? I don't specifically know the cases in which this would occur

Reply:
Yes, if not configured it will default to the existing permissive behaviour. This seems like the best option, since data requests are supposed to be open to any registered user, we just need to temporarily deal with our spam situation.